packages/core/src/components/fieldset/fieldset.stories.js

```diff
@@ -74,3 +75,19 @@ export const WithBorder = () => `
       </div>
     </fieldset>
 `;
+
+export const WithHelperText = () => `
+    <fieldset class='hds-fieldset'>
+      <legend class='hds-fieldset-legend'>
+        Applicant information
+      </legend>
+      <div style='display: grid; grid-gap: 12px 16px; grid-template-columns: repeat(auto-fill, minmax(245px, 1fr));'>
+       ${firstNameInput}
+       ${lastNameInput}
+       </div>
+      <div style='margin-top: 12px'>
+       ${socialSecurityNumberInput}
+      </div>
+      ${getHelperText()}
```

Comment:
I guess we need to ask accessibility experts how we connect the helper-text into the fieldset. Or does it even need to be connected, is it enough that it is inside the element?

Reply:
It's enough that text is inside <fieldset> tags